README.md

```diff
@@ -33,6 +33,10 @@ Mixins can be imported directly
 import { SvgIcon } from '@hypothesis/frontend-shared';
 ```
 
+### Additional documentation
+
+[Development guide](https://github.com/hypothesis/frontend-shared/blob/main/docs/developing.md)
```

Comment:
You can use a relative link here (`docs/developing.md`). See https://github.blog/2013-01-31-relative-links-in-markup-files/.